.changeset/serious-starfishes-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```

Comment:
You are fixing small typo in this PR so it's a "patch" - could you change it?

Reply:
Okay